w3-blob.md

```diff
@@ -375,13 +466,36 @@ Receipt MUST not have any effects.
 Location claim represents commitment from the issuer to the audience that
 content matching the `content` [multihash] can be read via HTTP [range request]
 
+### Location Claim Delegation Example
+
+```js
+{
+  "iss": "did:web:web3.storage",
+  "aud": "did:key:zAlice",
+
+  "cmd": "/assert/location",
+  "sub": "did:web:web3.storage",
+  "pol": [
+    // multihash must match be for the blob uploaded
+    ["==", ".content", { "/": { "bytes": "mEi...sfKg" } }],
```

Comment:
I actually now realize that content claims won't work with content?

Reply:
Why not ?

Reply:
Everything is supposed to be a link https://github.com/web3-storage/content-claims/blob/main/packages/core/src/capability/assert.js#L15 
we need to raise this with rest of the team, probably a good opportunity to already rename to commitment?

Reply:
I think there are several options we could consider

1. Make that field a type union so it it could be either CID or multihash.
1. Encode multihash as CID with raw codec
1. Use different cmd name
1. Upgrade path

We could also just use a different field name and deprecate old one. For what it's worth linked capability uses `location` field as opposed to `url` and that could be used as signal to differentiate between two.

I have no strong feeling about which option to go with, I just don't want to tie us to the decisions we made in the past

Reply:
```js
class ContentLocationClaim {
   constructor(model) {
        this.model = model
   }
   get content() {
      if (this.model.content instanceof Uint8Array) {
          return Link.create(RAW, Digest.decode(htis.model.content))
      } else {
          return this.model.content
      }
   }
   // ....
}

Reply:
We agreed to keep data model as defined here and decide how to encode this store separately